frontend/src/metabase-lib/queries/StructuredQuery.ts

```diff
@@ -737,10 +707,6 @@ class StructuredQuery extends AtomicQuery {
     return this._updateQuery(Q.removeField, arguments);
   }
 
-  clearFields() {
```

Comment:
We can remove it in `Join` too

Reply:
Great catch.
I think I found another thread related to that in the `Join` class, so let me follow up in a separate PR.